proto/osmosis/superfluid/superfluid.proto

```diff
@@ -81,3 +81,15 @@ message LockIdIntermediaryAccountConnection {
 }
 
 message UnpoolWhitelistedPools { repeated uint64 ids = 1; }
+
+message ConcentratedPoolUserPositionRecord {
+  string validator_address = 1;
```

Comment:
While I could have extracted the validator address one level higher, since we currently only allow a single validator to be staked to per pool, I decided to add it at a per-position level in case we decide to change this in the future and reduce the API breaks.

Reply:
What if we call this `UserDelegatedPosition` or the like